atomic_reactor/utils/cachi2.py

```diff
@@ -110,18 +110,25 @@ def gen_dependency_from_sbom_component(sbom_dep: Dict[str, Any]) -> Dict[str, Op
             heuristic_type = request_type
             break
 
+    pkg_dot_path = ("golang", "gem")
+
     version = (
         # for non-registry dependencies cachito uses URL as version
         purl.qualifiers.get("vcs_url") or
         purl.qualifiers.get("download_url") or
         # for local dependencies Cachito uses path as version
-        (f"./{purl.subpath}" if purl.subpath and purl.type == "golang" else None) or
-        (f"file:{purl.subpath}" if purl.subpath and purl.type != "golang" else None) or
+        (f"./{purl.subpath}" if purl.subpath and purl.type in pkg_dot_path else None) or
+        (f"file:{purl.subpath}" if purl.subpath and purl.type not in pkg_dot_path else None) or
```

Comment:
Did Cachito use `./path` instead of `file:path` for ruby deps? I don't quite remember

Reply:
yes, tested for you in integration tests